### ISSUE TITLE
Fix French keyboard information on the aria-keyshortcuts page

### ISSUE THREAD

**Issue by CyrilleB79** [mdn/content#18580]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-keyshortcuts

### What specific section or headline is this issue about?

Consider language and keyboard differences

### What information was incorrect, unhelpful, or incomplete?

The following sentence is wrong:
> For example, numbers, when the keyboard language preference is set to French, use the Control key.

### What did you expect to see?

The sentence should be corrected as follows:
> For example, numbers, when the keyboard language preference is set to French(France) , use the Shift key.

I.e. "Control" replaced by "Shift".
I have also defined more precisely the keyboard layout as "French (France)" in this sentence since not all French keyboard layouts use shift for number keypresses, as reported in https://github.com/mdn/content/issues/18580#issuecomment-1191342902.

### Do you have any supporting links, references, or citations?

In addition to my daily experience:
* https://en.wikipedia.org/wiki/AZERTY
  See the image of the keyboard
* https://www.thelocal.fr/20160120/what-annoys-expats-the-most-about-the-french-keyboard/
  > Numbers are not prioritized
Writing the numbers on a French keyboard requires using the shift key each time.

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/accessibility/aria/attributes/aria-keyshortcuts`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-keyshortcuts
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/accessibility/aria/attributes/aria-keyshortcuts/index.md
* Last commit: https://github.com/mdn/content/commit/0df41fd3125fba4a65e3900e0a6effb7072144c9
* Document last modified: 2022-06-28T05:17:50.000Z

</details>

**Comment by teoli2003** [comment]
Yes, Azerty uses Shift, but how does it relate to the French language itself?

For example, Swiss french people don't use the French keyboard, but a QWERTZ one with accented letters. On it, you don't need to press <kbd>Shift</kbd> for the digits.

So setting, or considering, the _language_ only is not enough.

**Comment by CyrilleB79** [comment]
> Yes, Azerty uses Shift, but how does it relate to the French language itself?
> 
> For example, Swiss french people don't use the French keyboard, but a QWERTZ one with accented letters. On it, you don't need to press Shift for the digits.
> 
> So setting, or considering, the _language_ only is not enough.

The goal of this paragraph is to indicate that for some locale of keyboard layout, numbers are not typed by a single keypress but with the help of a modifier. Unfortunately, the example was mentioning "Control" whereas "Shift" is used instead, in the French (France) keyboard layout.

In accordance with your comment, I have updated the initial description to indicate "Franch (France)" instead of "French" only.

Let me know if it addresses your concerns or if I should reword something else. Thanks.
